content/pl/concepts/overview/components.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Składniki Kubernetes
+content_template: templates/concept
+weight: 20
+card: 
+  name: concepts
+  weight: 20
+---
+
+{{% capture overview %}}
+W wyniku instalacji Kubernetes otrzymujesz klaster.
+
+{{< glossary_definition term_id="cluster" length="all" prepend="Klaster to">}}
+
+W tym dokumencie opisujemy składniki niezbędne do zbudowania kompletnego, poprawnie działającego klastra Kubernetes.
+
+Poniższy rysunek przedstawia klaster Kubernetes i powiązania pomiędzy jego różnymi częściami składowymi.
+
+![Składniki Kubernetes](/images/docs/components-of-kubernetes.png)
```

Comment:
Are we going also to translate images?

Reply:
Let's leave image translation to the next milestone.